integration-cli/docker_cli_commit_test.go

```diff
@@ -116,9 +116,9 @@ func (s *DockerSuite) TestCommitChange(c *check.C) {
 		"--change", "ENV test 1",
 		"--change", "ENV PATH /foo",
 		"--change", "LABEL foo bar",
-		"--change", "CMD [\"/bin/sh\"]",
+		"--change", `"CMD [""/bin/sh""]"`,
```

Comment:
Thanks @LK4D4 for the review. It looks like cobra requires quotes from outside (as there is a space in `CMD` and `[]`. But then to escape the quotes inside the quotes, a double quotes will be needed.

Therefore the whole argument has to be (single quotes outside and double quotes inside):
```
"CMD [""/bin/sh""]"
```

Otherwise it will return error:
```
invalid argument "CMD [\"/bin/sh\"]" for --change: line 1, column 5: bare " in non-quoted-field
```